extensions/apiconnect/README.md

```diff
@@ -3,7 +3,9 @@
 This module extends LoopBack with the ability to integrate with
 [IBM API Connect](https://www.ibm.com/cloud/api-connect). It comes with an
 `ApiConnectComponent` that adds an `OASEnhencer` extension to contribute
```

Comment:
While you're here, it would be good to fix this typo as well.

```suggestion
`ApiConnectComponent` that adds an `OASEnhancer` extension to contribute
```